staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -269,12 +271,18 @@ func (tc *patchTestCase) Run(t *testing.T) {
 
 		resultPod := resultObj.(*api.Pod)
 
-		// TODO: In the process of applying patches, we are calling
```

Comment:
this is restoring the unit test to the state prior to https://github.com/kubernetes/kubernetes/commit/e34e1abe3345939268c1fdbc05e2d933396ab2fe#diff-517d1b81963bbc7c9b0a16e6eb3c0e2f